### ISSUE TITLE
Using PHP 8 Beta 1 results in fatal Symfony unknown parameter error trying to view table structure

### ISSUE THREAD

**Issue by W1DJM** [phpmyadmin/phpmyadmin#16301]
```
[11-Aug-2020 10:14:15 America/New_York] PHP Fatal error:  Uncaught Error: Unknown named parameter $create_add_field in D:\phpMyAdmin\vendor\symfony\dependency-injection\ContainerBuilder.php:1144
Stack trace:
#0 D:\phpMyAdmin\vendor\symfony\dependency-injection\ContainerBuilder.php(1144): ReflectionClass->newInstanceArgs(Array)
#1 D:\phpMyAdmin\vendor\symfony\dependency-injection\ContainerBuilder.php(618): Symfony\Component\DependencyInjection\ContainerBuilder->createService(Object(Symfony\Component\DependencyInjection\Definition), Array, true, 'PhpMyAdmin\\Cont...')
#2 D:\phpMyAdmin\vendor\symfony\dependency-injection\ContainerBuilder.php(558): Symfony\Component\DependencyInjection\ContainerBuilder->doGet('PhpMyAdmin\\Cont...', 1)
#3 D:\phpMyAdmin\tbl_structure.php(67): Symfony\Component\DependencyInjection\ContainerBuilder->get('PhpMyAdmin\\Cont...')
#4 {main}
  thrown in D:\phpMyAdmin\vendor\symfony\dependency-injection\ContainerBuilder.php on line 1144
```

### To Reproduce

Attempt to look at the Structure tab of any database and you will get the error detailed above.

### Expected behavior

I expect the table structure to be shown.

### Server configuration

Windows 10 Pro
Apache 2.4.46
MariaDB 10.5.4
8.0 Beta 1
5.0.2

Fails with any browser.

**Comment by williamdes** [comment]
@fabpot how do I report this to https://github.com/symfony/dependency-injection ?

**Comment by fabpot** [comment]
This should be reported to github.com/symfony/symfony.

**Comment by williamdes** [comment]
Hi @W1DJM 
It seems to have been fixed in `symfony/symfony` (https://github.com/symfony/symfony/issues/38089#issuecomment-688286247)

@mauriciofauth should we bump the minor 4.x version we require on `QA_5_0` ?

**Comment by williamdes** [comment]
I believe, given the tags I can see on https://github.com/symfony/symfony/commit/8e34978e4e2c4bf7dac97e44e904368c4cfb54f8 that 5.0.3 release does have this change included.

**Comment by williamdes** [comment]
Confirmed, https://github.com/symfony/symfony/commit/8e34978e4e2c4bf7dac97e44e904368c4cfb54f8 was shipped into 5.0.3 as you can see on https://github.com/code-lts/phpmyadmin-releases/blame/upstream/version/source/5.0.3/vendor/symfony/dependency-injection/ContainerBuilder.php#L1144

**Comment by ahmedfoad59** [comment]
i need to simpel Answer for this problem
